applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities/element_setup_utilities.cpp

```diff
@@ -0,0 +1,146 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "element_setup_utilities.h"
+#include "custom_elements/U_Pw_small_strain_element.hpp"
+
+#include "custom_elements/plane_strain_stress_state.h"
+#include "geometries/triangle_2d_3.h"
+
+#include <custom_elements/three_dimensional_stress_state.h>
+#include <geometries/tetrahedra_3d_10.h>
+#include <geometries/triangle_2d_10.h>
+#include <geometries/triangle_2d_15.h>
+#include <geometries/triangle_2d_6.h>
```

Comment:
I would suggest to make this a single block of `"include`'s with a consistent way of enclosing the relative path to the header files (running `kp format` may change the line order):
```suggestion
#include "element_setup_utilities.h"
#include "custom_elements/U_Pw_small_strain_element.hpp"
#include "custom_elements/plane_strain_stress_state.h"
#include "geometries/triangle_2d_3.h"
#include "custom_elements/three_dimensional_stress_state.h"
#include "geometries/tetrahedra_3d_10.h"
#include "geometries/triangle_2d_10.h"
#include "geometries/triangle_2d_15.h"
#include "geometries/triangle_2d_6.h"
```

Reply:
Done